tempesta_fw/http.c

```diff
@@ -390,6 +389,26 @@ tfw_http_send_504(TfwHttpReq *req, const char *source, const char *reason)
 }
 
 /*
+ * SKB data is needed for calculation of a cache key from fields of
+ * a request. It's also needed when a request may need to be re-sent.
+ * In all other cases it can just be passed to the network layer.
+ */
+static inline void
+tfw_http_req_init_ss_flags(TfwSrvConn *srv_conn, TfwHttpReq *req)
+{
+	TfwSrvGroup *sg = ((TfwServer *)(srv_conn->peer))->sg;
+	if (tfw_cache_msg_cacheable(req) || (req->retries < sg->max_refwd))
```

Comment:
I think there is no need to optimize this rare case and we can just always copy skb. It's bad, but we'll address this in #391 and #488.